### ISSUE TITLE
ProcessingError for secondaryVisibilityStore 

### ISSUE THREAD

**Issue by DodgeCamaro** [alexandrevilain/temporal-operator#486]
When I've created secondaryVisibilityStore for Opensearch, I've got this error

```
2023-08-29T08:28:39Z	DEBUG	events	Job.batch "temporal-update-secondary-visibility-schema-v-1-21-4-elasticsearch" is invalid: spec.template.labels: Invalid value: "temporal-update-secondary-visibility-schema-v-1-21-4-elasticsearch": must be no more than 63 characters	{"type": "Warning", "object": {"kind":"TemporalCluster","namespace":"default","name":"temporal","uid":"c58551c2-6abc-47cc-bbde-8d6034aa33c4","apiVersion":"temporal.io/v1beta1","resourceVersion":"114052936"}, "reason": "ProcessingError"}
2023-08-29T08:26:36Z	ERROR	Can't reconcile persistence	{"controller": "temporalcluster", "controllerGroup": "temporal.io", "controllerKind": "TemporalCluster", "TemporalCluster": {"name":"temporal","namespace":"default"}, "namespace": "default", "name": "temporal", "reconcileID": "1293bee0-2fdc-4d75-bfc6-31a69d5c2cf4", "error": "Job.batch \"temporal-update-secondary-visibility-schema-v-1-21-4-elasticsearch\" is invalid: spec.template.labels: Invalid value: \"temporal-update-secondary-visibility-schema-v-1-21-4-elasticsearch\": must be no more than 63 characters"}
```

**Comment by alexandrevilain** [comment]
Thanks for the report @DodgeCamaro ! I'll try to fix this asap and to release 0.15.3

**Comment by alexandrevilain** [comment]
@DodgeCamaro 0.15.3 is now released with the fix. Hope it will work well.
